Sources/JXKit/JXContext.swift

```diff
@@ -4,24 +4,63 @@ import JavaScriptCore
 #else
 import CJSCore
 #endif
-#if canImport(FoundationNetworking)
-import FoundationNetworking
+#if canImport(Foundation)
+import Foundation
 #endif
 
 /// A JavaScript execution context. This is a cross-platform analogue to the Objective-C `JavaScriptCore.JSContext`.
 ///
 /// This wraps a `JSGlobalContextRef`, and is the equivalent of `JavaScriptCore.JSContext`
 open class JXContext {
+    /// Context configuration.
+    public struct Configuration {
+        /// Whether scripts evaluated by this context should be assessed in `strict` mode. Defaults to `true`.
+        public var strict: Bool
+        
+        /// Whether hot reloading of JavaScript script resources is enabled.
+        public var isHotReloadEnabled: Bool {
+#if canImport(Foundation)
+            return scriptLoader.didChange != nil
```

Comment:
As you'll see in JXScriptLoader, I'm currently basing whether we attempt to hot reload on whether the ScriptLoader vends a non-nil change listener collection. I bundled it into ScriptLoader because I feel like loading and monitoring are going to be pretty intimately linked. And having a single property (as opposed to e.g. a separate protocol) lets you create one ScriptLoader implementation that conditionally vends nil or non-nil depending on environment or configuration or whatever.